### ISSUE TITLE
Accessibility violations

### ISSUE THREAD

**Issue by ghost** [securingsincity/react-ace#374]
# Problem

While testing my app for accessibility compliance, I found the following violations related to the Ace editor.

1 - Color contrast issues using the `monokai` theme.
<img width="228" alt="screen shot 2018-02-09 at 11 31 30 am" src="https://user-images.githubusercontent.com/17085131/36038148-ddc81c00-0d8c-11e8-811c-35e15e916bf8.png">

<img width="870" alt="2429c856-0a7c-11e8-93f4-b2d1f0087f4c" src="https://user-images.githubusercontent.com/17085131/36038094-a8a7a7b6-0d8c-11e8-8b33-9668f89a5e5d.png">

2 - `Textarea` missing label
<img width="1383" alt="screen shot 2018-02-09 at 11 32 40 am" src="https://user-images.githubusercontent.com/17085131/36038187-fde2b8f6-0d8c-11e8-8d70-6f4d9fa51391.png">

**Comment by securingsincity** [comment]
Hi all given it's been quite some time and no activity on this issue, I am closing it.
